swim.go

```diff
@@ -219,3 +224,68 @@ func (s *SWIM) handlePbk(piggyBack *pb.PiggyBack) {
 		s.pbkStore.Push(*piggyBack)
 	}
 }
+
+// handleIndirectPing receives IndirectPing message, so send the ping message
+// to target member, if successfully receives ACK message then send back again
+// ACK message to member who sent IndirectPing message to me.
+// If ping was not successful then send back NACK message
+func (s *SWIM) handleIndirectPing(msg pb.Message) {
+	seq := msg.Seq
```

Comment:
I think we can change 
`func (s *SWIM) handleIndirectPing(msg pb.Message) {` to 
`func (s *SWIM) handleIndirectPing(msg pb. Message_IndirectPing) {`